tests/functional/syntax/test_codehash.py

```diff
@@ -28,8 +27,7 @@ def foo3() -> bytes32:
 def foo4() -> bytes32:
     return self.a.codehash
     """
-    settings = Settings(optimize=optimize, experimental_codegen=experimental_codegen)
-    compiled = compile_code(code, output_formats=["bytecode_runtime"], settings=settings)
+    compiled = compile_code(code, output_formats=["bytecode_runtime"], settings=compiler_settings)
```

Comment:
we can leave out the `settings=compiler_settings` right? since that falls back to the global settings